### ISSUE TITLE
Document as many board ids as possible (and assign them to the respective board types)

### ISSUE THREAD

**Issue by zecoxao** [andy-man/ps4-wee-tools#10]
Thanks to master hacker flatz (and this not only applies to ps5 cp box but to ps5 sflash and ps4 sflash as well) it has been discovered that 0x1C4000 (or the start of nvs) size 8 corresponds to board id. my suggestion would be to note down these digits and make notes to which type of board they have (region, model, hw info, etc) and draw a pattern of sorts. i suggest putting pictures here with the latest wee tools and respective board id pasted as text. also, if possible adding support for displaying board id would be great

**Comment by zecoxao** [comment]
Board ID 01 01 01 01 01 01 01 01 -> Syrup Board

**Comment by andy-man** [comment]
Good idea, I've added board_id info + mb version suggestion to info screen (since 0.8.7)

CUH-7115B B01 [05:02:02:01:01:05:05:01] NVB-003

**Comment by MohammedTayyab6339** [comment]
CUH-7115B [NVB-003]

**Comment by DonZott** [comment]
CUH-2116A B01 [03:02:05:01:01:03:05:01] SAE-002

**Comment by tekaikko** [comment]
CUH-2216B B01 [03:02:06:01:01:07:05:01] SAF-006
Region: [16] Europe / Middle East / Africa
Southbridge: Baikal B1
Torus (Wifi): Version 3

**Comment by tikyuchan** [comment]
SKU / Board ID   : CUH-7200B B01 [05:02:03:01:01:07:05:01] ~NVG-004
 Region           : [00] Japan
 Southbridge      : [0x24, 0x00]
 Torus (WiFi)     : Version 3

**Comment by MSXCelulares** [comment]
Hello my friend, with the data obtained so far I came to the conclusion that the motherboard is made up of Prefix + Suffix - Revision, where:
0x1c4000 - is prefix (02=CV, 03=SA, 04=HV, 05=NV)
0x1c4001 - is Non-Retail or Retail (01=N-r, 02=R)
0x1c4002 - is suffix:
if prefix=02: suffix=N, e.g.: CVN-k04
if prefix=03: suffix=Alphabet seq ...
if prefix=05: suffix=A,B or G ...
The review is still unknown to me.
The addresses:
0x1c4006 and 0x1c4007 compared to a BeW report as far as I could analyze are flags to check if the HDD was replaced or is original from the factory.
Below is a copy of the sFlash.py file with some changes I made and they worked perfectly with the dumps.bin I have
[dados de placas.txt](https://github.com/andy-man/ps4-wee-tools/files/12702984/dados.de.placas.txt)
[sflash.zip](https://github.com/andy-man/ps4-wee-tools/files/12702988/sflash.zip)
![Test1_v0 8 7b](https://github.com/andy-man/ps4-wee-tools/assets/96247543/9eef228f-3a25-4dcb-96e0-3c4e090733f2)
![Test2_v0 8 7b](https://github.com/andy-man/ps4-wee-tools/assets/96247543/99e03713-d8b0-4d4b-9e2a-3b8fe0f702f1)

**Comment by saqiba609** [comment]
SKU / Board ID   : CUH-7216B B01 [05:02:03:01:01:05:05:01] NVG-001
 Region           : [16] Europe / Middle East / Africa - [Retail]
 Southbridge      : Belize 2 A0 [CXD90046] [2A:2B]
 Torus (WiFi)     : V3 - MediaTek MT7667BSN [0x30] - 2.4+5Ghz [0x8C]
